cfme/provisioning.py

```diff
@@ -35,6 +40,137 @@ def select_security_group(sg):
     sel.sleep(1)
 
 
+class ProvisioningForm(BaseLoggedInPage):
+    @View.nested
+    class request(Tab):  # noqa
+        TAB_NAME = 'Request'
+        email = Input('requester__owner_email')
```

Comment:
Look for the definition of Tab class and add `INDIRECT = True`.

@mshriver  I have it in my AE PR but if anyone beats me to it, I am okay with that :)

INDIRECT causes to ignore the root locator for given view class, which is required for Tab as the ROOT contains only the tab on top but the actual content is located somewhere else :)